packages/client/utils/reactjiPickerUtils.ts

```diff
@@ -0,0 +1,28 @@
+const DEFAULT = {
+  heart: 9,
+  tada: 8,
+  smile: 7,
+  rocket: 6,
+  fire: 5,
+  white_check_mark: 4,
+  confused: 3,
+  cry: 2,
+  x: 1
+}
+
+const getRecent = () => {
+  const localStorageRecent = localStorage.getItem('recent')
+  return localStorageRecent === null ? DEFAULT : JSON.parse(localStorageRecent)
+}
+
+const handleRecent = (emoji) => {
+  const frequentlyUsed = getRecent()
+  const {id} = emoji
+  frequentlyUsed[id] || (frequentlyUsed[id] = 0)
+  frequentlyUsed[id] += 1
+  localStorage.setItem('recent', JSON.stringify(frequentlyUsed))
+}
+
+const getRecentArray = () => Object.keys(getRecent())
```

Comment:
This will not order the array by usage. Also it's not returning all keys if only some were set. It would be best to order all keys with usage by their order and then followed by the remaining keys from `DEFAULT_ORDER`. Since you're using this as learning exercise I leave that for you ;-)

Naming could also be improved, like `getRecentOrder`.